src/components/APIStructurePreview.tsx

```diff
@@ -94,23 +94,23 @@ interface CardProps {
   position: React.CSSProperties;
 }
 
-const Card = (props: CardProps) => (
-  <article
-    ref={props.innerRef}
-    id="structures-tooltip"
-    role="tooltip"
-    className={clsx('alert', 'alert--info', 'shadow--md', styles.card)}
-    style={props.position}
-  >
-    <div className="card__body">
-      <ReactMarkdown
-        allowedElements={['h1', 'ul', 'li', 'code', 'a', 'em']}
-        remarkPlugins={[apiLabels]}
-      >
-        {decodeURIComponent(props.content)}
-      </ReactMarkdown>
-    </div>
-  </article>
-);
+const Card = (props: CardProps) => {
+  return (
+    <article
+      ref={props.innerRef}
+      id="structures-tooltip"
+      role="tooltip"
+      className={clsx('alert', 'alert--info', 'shadow--md', styles.card)}
+      style={props.position}
+    >
+      <div
+        className="card__body"
+        dangerouslySetInnerHTML={{
+          __html: toHtml(toHast(JSON.parse(props.content))),
```

Comment:
[According to the `mdast` docs on security](https://github.com/syntax-tree/mdast?tab=readme-ov-file#security), we should run the result of `toHast` through `hast-util-sanitize` before running it through `toHtml`.

Reply:
[a149300](https://github.com/electron/website/pull/690/commits/a1493006b38992102b2337694aa6d7f805175b85)